xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1387,6 +1387,110 @@ class FlashAttentionPagedAttention : public MultiHeadedAttentionTest {
   }
 };
 
+class FlashAttentionFlexAttention : public MultiHeadedAttentionTest {
+ protected:
+  const std::string
+  GetModuleFlash_Attention_Flex_Attention_Soft_Capping_BF16() {
+    const std::string hlo_text = R"(
```

Comment:
Please make the test HLO more human readable:
* Remove default layouts
* Use readable names like `zero = f32[] constant(0)` (rather than `%constant.8`) or `add` rather than `%region_1.26`
*